base/expvar.go

```diff
@@ -82,6 +82,10 @@ const (
 	StatKeyChannelCachePendingQueries = "chan_cache_pending_queries"
 	StatKeyNumSkippedSeqs             = "num_skipped_seqs"
 	StatKeyAbandonedSeqs              = "abandoned_seqs"
+	StatKeyMaxCached                  = "max_cached"
+	StatKeyMaxStableCached            = "max_stable_cached"
+	StatKeyNumSkipped                 = "num_skipped"
```

Comment:
Above we have `num_skipped_seqs`, which I suspect is the monotonic count of ones that have been added to the skipped queue - need to think about how to name these consistently and non-confusingly.

Maybe `num_currently_*_seqs`?